cmd/vgmanager/main.go

```diff
@@ -75,8 +75,9 @@ func main() {
 	}
 
 	if err = (&vgmanager.VGReconciler{
-		Client: mgr.GetClient(),
-		Scheme: mgr.GetScheme(),
+		Client:   mgr.GetClient(),
+		Scheme:   mgr.GetScheme(),
+		Nodename: os.Getenv("NODE_NAME"),
```

Comment:
Where is set this env. variable?

Reply:
[here](https://github.com/red-hat-storage/lvm-operator/blob/b76aeb2f19aa0aceacf955c56eb70d7c661f468e/controllers/vgmanager_daemonset.go#L157).